content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -316,11 +316,9 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 ### Volume Mode
 
-{{< feature-state for_k8s_version="v1.13" state="beta" >}}
-
 Prior to Kubernetes 1.9, all volume plugins created a filesystem on the persistent volume.
-Now, you can set the value of `volumeMode` to `block` to use a raw block device, or `filesystem`
-to use a filesystem. `filesystem` is the default if the value is omitted. This is an optional API
+Now, you can set the value of `volumeMode` to `Block` to use a raw block device, or `Filesystem`
```

Comment:
hello @jsafrane , If the state is changing for this feature, would you update the feature gate table (move this feature to the second table),
https://5dc002804c2758000842488e--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Moved to `Feature gates for graduated or deprecated features` and added GA in 1.17.